common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/NativeAuthOAuth2StrategyFactory.kt

```diff
@@ -40,11 +41,16 @@ class NativeAuthOAuth2StrategyFactory {
             return NativeAuthOAuth2Strategy(
                 strategyParameters = strategyParameters,
                 config = config,
-                signInInteractor = SignInInteractor(
+                signUpInteractor = SignUpInteractor(
                     httpClient = UrlConnectionHttpClient.getDefaultInstance(),
                     nativeAuthRequestProvider = NativeAuthRequestProvider(config = config),
                     nativeAuthResponseHandler = NativeAuthResponseHandler()
                 ),
```

Comment:
Nit: we can switch the order of the interactors passed here to match the constructor. This is completely inconsequential imo though, feel free to ignore

Reply:
Done